netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -31,11 +31,8 @@
 
 package io.grpc.transport.netty;
 
-import static io.netty.util.CharsetUtil.UTF_8;
```

Comment:
ugh, sorry, I should have audited the imports diff, I was having trouble with those before.